packages/node_modules/@ciscospark/react-component-space-item/src/index.test.js

```diff
@@ -44,6 +44,31 @@ describe('SpaceItem component', () => {
     expect(component).toMatchSnapshot();
   });
 
+  it('correctly renders an active space', () => {
+    renderer.render(
+      <SpaceItem
+        activityText="I'm active!"
+        id="active-space-id"
+        lastActivityTime="3:05 PM"
+        latestActivity={{
+          actorName: 'Andrew',
+          type: 'post'
+        }}
+        activeSpaceId="active-space-id"
+        onCallClick={onClick}
+        onClick={onClick}
+        name="Webex User"
+        teamColor="blue"
+        teamName="Best Team"
+      />
+    );
+    const component = renderer.getRenderOutput();
+    console.log(component.props.className)
+    expect(component.props.className.includes('active')).toBeTruthy();
```

Comment:
This is the test we don't need due to the snapshot being valid.

Reply:
When I originally created the snapshot, it didn't have the `active` class. So my snapshot test was passing but it didn't really have everything I wanted. How do you know that the snapshot represents what you're expecting? Just manually validate and once you do that, you know it's good and you keep the snapshot?

Reply:
Gotcha, yea, when a snapshot is initially created, you have to manually verify it looks like you want it to.